generators/server/templates/_pom.xml

```diff
@@ -941,9 +963,6 @@
                     Enable the line below to have remote debugging of your application on port 5005
                     <jvmArguments>-agentlib:jdwp=transport=dt_socket,server=y,suspend=n,address=5005</jvmArguments>
                     -->
-                    <arguments>
-                        <argument>--spring.profiles.active=dev</argument>
```

Comment:
Oh ok I guess I didnt think it out properly, in dev profile the default will kick in anyway and in prod profile the resources are served from target folder which will have the value replaced file am I right?